package.json

```diff
@@ -9,8 +9,8 @@
     "npm": ">=1.3.0"
   },
   "devDependencies": {
-    "eslint": "^8.34.0",
-    "puppeteer": "^19.7.0",
+    "eslint": "^8.36.0",
+    "puppeteer": "^18.2.1",
```

Comment:
`puppeteer` 19+ changed caching strategy:
```
Options: timeout=5000, inject="/home/runner/work/django/django/node_modules/grunt-contrib-qunit/chrome/bridge.js", urls=[], force=false, console, httpBase=false, summaryOnly=false
>> There was an error with headless chrome
Fatal error: Could not find Chromium (rev. 1108766). This can occur if either
 1. you did not perform an installation before running the script (e.g. `npm install`) or
 2. your cache path is incorrectly configured (which is: /home/runner/work/django/django/.cache/puppeteer).
For (2), check out our guide on configuring puppeteer at [https://pptr.dev/guides/configuration.](https://pptr.dev/guides/configuration.)
```
I don't know how to fix it (adding [`.puppeteerrc.cjs`](https://pptr.dev/guides/configuration#changing-the-default-cache-directory) didn't help). I think it's fine to downgrade it for now.

Reply:
Agreed. Merge the black change, and then we can look at the JS deps separately.

- [ ] Find new JS Hero 🦸‍♀️